.github/workflows/pr.yaml

```diff
@@ -115,7 +115,7 @@ jobs:
     uses: rapidsai/shared-workflows/.github/workflows/custom-job.yaml@branch-25.02
     with:
       build_type: pull-request
-      node_type: "gpu-v100-latest-1"
+      node_type: "gpu-l4-latest-1"
```

Comment:
It looks like these workflows are still pointing at `branch-25.02`, not `nvks-runners`

Reply:
Wow! Not sure how that occurred. Fixed in 17b5bcaa64c7ea364669b1a7fe4399aeed386693.